testing/internal/e2e/host/static/connect_ssh_test.go

```diff
@@ -94,6 +97,26 @@ func TestConnectTargetWithSshCli(t *testing.T) {
 	)
 	require.NoError(t, output.Err, string(output.Stderr))
 	t.Log("Successfully connected to target")
+
+	// Delete credential store
+	output = e2e.RunCommand(ctx, "boundary", "credential-stores", "delete", "-id", newCredentialStoreId)
+	require.NoError(t, output.Err, string(output.Stderr))
+	err = backoff.RetryNotify(
+		func() error {
+			output = e2e.RunCommand(ctx, "boundary", "credential-stores", "read", "-id", newCredentialStoreId, "-format", "json")
+			if output.Err == nil {
+				return errors.New("Deleted credential store can still be read")
+			}
```

Comment:
Do we also want to check that the eventual error is a 404 not found?

Reply:
@johanbrandhorst I tried adding something. I created a struct to interpret the stderror from the cli in order to get the status code. I didn't see one defined in the application code.